docs/source/how_to_extend.rst

```diff
@@ -46,7 +46,7 @@ Now that we have the new library set up to send requests, we just have to update
 
 .. code-block:: python
     :caption: response_retriever.py
-    :emphasize-lines: 7
+    :emphasize-lines: 12
```

Comment:
Same alignment problem here. It would be awesome if the editor checked this for us.

Reply:
Is it not showing up properly for you?  Just curious because it's showing up fine for me with the extra space.  I will correct though!